catenax/.htaccess

```
@@ -87,5 +87,9 @@ RewriteRule ^next/taxonomy(/|\.ttl)?$ https://raw.githubusercontent.com/catenax-
 # Rewrite rule to serve complete taxonomy
 RewriteRule ^taxonomy(/|\.ttl)?$ https://raw.githubusercontent.com/catenax-ng/product-ontology/v23.12/taxonomy.ttl [R=302,L]
 
+# ODRL Profile definition
+RewriteRule ^policy/$ https://raw.githubusercontent.com/catenax-eV/cx-odrl-profile/main/profile.ttl [R=302,L]
```

Did you want the similar optional trailing slash or .ttl extension like the other rules here?
```suggestion
RewriteRule ^policy(/|\.ttl)?$ https://raw.githubusercontent.com/catenax-eV/cx-odrl-profile/main/profile.ttl [R=302,L]
```

---

Thanks for the hint @davidlehn ! I have refined the rule to also allow without / and everything behind the / that someone who clicks on one of our identifers like:
https://w3id.org/catenax/policy/FrameworkAgreement
is also forwarded to the ttl file.

We might further refine this in the future to have dedicated html files for those, but right now, all should end up in the ttl file.